bundles/binding/org.openhab.binding.km200/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="km200"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>km200 Binding</name>
+    <description>The KM200 Binding is communicating with a Buderus Logamatic web KM200 / KM100 / KM50. It is possible to recive and send parameters like string or float values.</description>
+    <author>Markus Eckhardt</author>
+    
+    <service-id>org.openhab.binding.km200</service-id>
```

Comment:
The `<service-id>` needs to be identical to the service PID defined [here](https://github.com/openhab/openhab1-addons/blob/master/bundles/binding/org.openhab.binding.km200/OSGI-INF/km200binding.xml#L21), so it needs to be `org.openhab.km200`.